### PR TITLE
remove bhc

### PR DESCRIPTION
**Pull request by mfshao**
Remove BHC from schema validation scripts

### CODE REVIEW

File: testSchema.sh [hunk 1]
```diff
@@ -6,11 +6,11 @@ set -e
 declare -a testCases
 testCases=(
 bpa "data.bloodpac.org"
-bhc "data.braincommons.org"
-gtex "dcp.bionimbus.org"
+gtex "gen3.biodatacatalyst.nhlbi.nih.gov"
```
Comment (williamhaley) [inline testSchema.sh:9]: Just confirming that this `gtex` change was intentional 👀 while removing the `bhc` stuff

**Review by williamhaley — APPROVED** [review]
lgtm. One comment on a change that may be unrelated

File: testSchema.sh (hunk 1 above)
Comment (mfshao) [reply testSchema.sh:9]: Yeah I feel we may need it. Because we've changed gtex to bdcat? If not I'll just change it back 😅

Comment (williamhaley) [reply testSchema.sh:9]: 👍

**Pull request merged**